configs/mdanalysis.json

```diff
@@ -23,10 +23,10 @@
     "lvl2": "[itemprop='articleBody'] > .section h3, .page h3, .post h3, .body > .section h3",
     "lvl3": "[itemprop='articleBody'] > .section h4, .page h4, .post h4, .body > .section h4",
     "lvl4": "[itemprop='articleBody'] > .section h5, .page h5, .post h5, .body > .section h5",
-    "text": "[itemprop='articleBody'] > .section p, .page p, .post p, .body > .section p, [itemprop='articleBody'] > .section li, .page li, .post li, .body > .section li"
+    "text": "[itemprop='articleBody'] > .section p, .page p, .post p, .body > .section p, [itemprop='articleBody'] > .section li, .page li, .post li, .body > .section li, [itemprop='articleBody'] > .section dt, .body > .section dt, [itemprop='articleBody'] > .section pre, .page pre, .post pre, .body > .section pre, "
```

Comment:
FYI, having this number of selectors could add a lot of noise to the search results and hide other relevant results. A new crawl will be started right after the merge, don't hesitate to revert if you're not happy with the results!

Reply:
Thank you for the comment @shortcuts . 

Most of the selectors were added earlier by `@s-pace` to properly index our sites. If adding dt and pre to this list creates too much noise we'll take you up on your offer to revert.

Thank you for the opportunity to try it out.

I _hope_ that this also somehow solves #4699 .